app/models/media.rb

```diff
@@ -237,7 +237,7 @@ def absolute_url(path = '')
     if path =~ /^https?:/
       path
     elsif path =~ /^\/\//
-      self.parse_url(self.url).scheme + path
+      "#{self.parse_url(self.url).scheme}:#{path}"
```

Comment:
The interpolation was just to add the ":".. But I'll change it to be similar to the other line